Libs/MRML/Core/vtkDataFileFormatHelper.cxx

```diff
@@ -55,6 +55,7 @@ ITKImageFileFormat FileFormatTable[] =
   {"MGHImageIO", "Uncompressed FreeSurfer pixel data in binary", "MGH", ".mgh"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgz"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgh.gz"},
+  {"MINCImageIO", "Optional compression. Binary pixel data", "MINC", ".mnc"},
```

Comment:
Should be careful here, classically, .mnc.gz files are "MINC1" aka NetCDF files, which the ITK MINC IO can't open.

.mnc files are now typically MINC2 files which are HDF5 with internal zlib compression built into the filespec.